components/Chips/src/MDCChipCollectionViewFlowLayout.m

```diff
@@ -31,7 +39,8 @@ @implementation MDCChipCollectionViewFlowLayout
       [NSMutableArray arrayWithCapacity:layoutAttributes.count];
 
   if (layoutAttributes.count > 0) {
-    UICollectionViewLayoutAttributes *attrs = layoutAttributes[0];
+    UICollectionViewLayoutAttributes *attrs = [layoutAttributes[0] copy];
+    attrs.frame = CGRectLeftAlign(attrs.frame);
```

Comment:
Do you need to reset the `origin.x` to zero even when the first element in the `rect` would not be the first element in the collection?